src/lib/support/jsontlv/TlvToJson.h

```diff
@@ -48,4 +48,15 @@ std::string PrettyPrintJsonString(const std::string & jsonString);
  */
 std::string MakeJsonSingleLine(const std::string & jsonString);
 
+/*
+ *
+ * The attribute/event TLV blob from the report during IM read/subscribe Interaction does not have a struct container
+ * and attribute/event tag, but the tlvToJson function expects the struct container and tag Id for the TLV payload.
```

Comment:
So this seems like a basic bug in TlvToJson: instead of being "convert whatever value I have here to a JSON value" it's a "convert this value _which is expected to be a TLV struct, but I am not going to tell you that in the documentation_ to a JSON object".

But why is it not a value-to-value conversion to start with, instead of adding these workarounds around it's API brokenness?